### PR TITLE
Unnesting front-end

### PR DESCRIPTION
**Pull request by LamarSage**
Related issue: #48 
Description of Changes Made: The front end was nested in an unnecessary way
Questions/Comments for Reviewer:

### CODE REVIEW

**Comment by LamarSage** [comment]
netlify was adjusted by Matt so that this change would work.

File: Pipfile [hunk 1]
```diff
@@ -4,6 +4,8 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
+djangorestframework = "*"
+django-environ = "*"
```
Comment (Underwaterr) [inline Pipfile]: This shouldn't be part of the commit?

File: back-end/blltn_brd/settings.py [hunk 2]
```diff
@@ -98,7 +98,6 @@
     }
 }
 
-
```
Comment (Underwaterr) [inline back-end/blltn_brd/settings.py:101]: Neither should this 😲

**Review by Underwaterr — CHANGES_REQUESTED** [review]
Clean up unnecessary file changes for this commit
